robot/basestation/static/js/rover.js

```diff
@@ -552,6 +553,49 @@ function gameLoop () {
       }
     }
 
+    // rear camera position servo (up/down, servo 1)
+    if ( (millisSince(lastRearPosServoCmd) > POSITION_SERVO_PERIOD) && !$('#servo-val').is(':focus') ) {
+      let returnVals = handlePositionServo (
+        rearTiltPwm, minRearTiltPwm, maxRearTiltPwm,
+        101, 98, // numpad '5' and '2'
```

Comment:
For future reference we should definitely avoid use of [magic numbers](https://en.wikipedia.org/wiki/Magic_number_(programming)) (the first meaning).

Reply:
@PeterGhimself I've highlighted that in my new issue https://github.com/space-concordia-robotics/robotics-prototype/issues/160

Thanks for bringing it up